### PR TITLE
US_ND updates

### LINKED ISSUE #1566: US_ND FE adjustments
**What needs to be done? Why does it need to be done?**
- [x] Facility names are long - need to adjust the facility charts and facility filter

- [ ] There are two profiles on profile page when currentTenantId is US_ND

- [x] add `total count` to the metric object instead of pulling it from each individual record

- [x] adjust time_periods to not be inclusive

MVP feature set difference:
Prison
- [x] Filters: Gender
Prison to Supervision
- [ ] Charts: List of people released to supervision
Supervision to Prison
- [ ] Filters: Offense Type
- [ ] Charts: Admissions from supervision by officer
Supervision to Liberty
- [x] Filters: District, Age, Race

**Additional context**
Add any other context about the task here.

### PR DESCRIPTION
**Pull request by jovergaag**
## Description of the change

Several updates to support US_ND

- various UI tweaks
- Updates copy
- Fix Methodology page header
- Update Pathways tooltip props
- Use location name in location chart tooltips
- Adjust LH panel width
- Updates supervision to liberty filters

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes  #1566

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally
- [ ] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/utils/filterOptions.ts [hunk 1]
```diff
@@ -433,6 +457,147 @@ export const MePopulationFilterOptions: PopulationFilters = {
...
+      { label: "Minot Office", value: "Minot Office" },
+      { label: "Washburn Office", value: "Washburn Office" },
+      { label: "Watford City Office", value: "Watford City Office" },
+      { label: "Mandan Office", value: "Mandan Office" },
+      { label: "Oakes Office", value: "Oakes Office" },
+      { label: "Grafton Office", value: "Grafton Office" },
+      { label: "Grand Forks Office", value: "Grand Forks Office" },
+      { label: "Wahpeton Office", value: "Wahpeton Office" },
+      { label: "Williston Office", value: "Williston Office" },
+      { label: "Rolla Office", value: "Rolla Office" },
```
Comment (Catacola) [inline src/core/utils/filterOptions.ts:561]: Out of scope for this PR, but worth noting as we build this out.

I've noticed there's a lot of repeated label/value pairs. What do you think of making `label` optional and using the value of `value` if `label` is unset?

**Review by Catacola — APPROVED** [review]
Thanks for all the cleanup

File: src/core/CoreStore/FiltersStore.ts [hunk 2]
```diff
@@ -178,4 +179,12 @@ export default class FiltersStore {
       )?.label || ""
     );
   }
+
+  getLocationName(
+    filterType: keyof PopulationFilters,
+    locationValue: string
+  ): string | undefined {
+    const { locationNameMap } = this.filterOptions[filterType];
+    return locationNameMap ? locationNameMap[locationValue] : undefined;
```
[the remark this answers is not in the document]
Comment (jovergaag) [reply src/core/CoreStore/FiltersStore.ts:188]: Thanks :) Old habits die hard.

File: src/core/utils/filterOptions.ts (hunk 1 above)
Comment (jovergaag) [reply src/core/utils/filterOptions.ts:561]: I will add that as a task in another issue. I would like to get this merged today. #1604

**Pull request merged**